<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Openbrowser]]>
    </line>
    <line>
      <![CDATA[Login]]>
    </line>
    <line>
      <![CDATA[testcreateCustomer]]>
    </line>
    <line>
      <![CDATA[logout]]>
    </line>
    <line>
      <![CDATA[closebrowser]]>
    </line>
  </reporter-output>
  <suite started-at="2022-09-02T21:13:51 IST" name="Default suite" finished-at="2022-09-02T21:15:07 IST" duration-ms="75592">
    <groups>
    </groups>
    <test started-at="2022-09-02T21:13:51 IST" name="Default test" finished-at="2022-09-02T21:15:07 IST" duration-ms="75592">
      <class name="com.actitime.CScript2.CustomerModule22">
        <test-method is-config="true" signature="openbrowser()[pri:0, instance:com.actitime.CScript2.CustomerModule22@25084a1e]" started-at="2022-09-02T21:13:51 IST" name="openbrowser" finished-at="2022-09-02T21:13:52 IST" duration-ms="1272" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Openbrowser]]>
            </line>
          </reporter-output>
        </test-method> <!-- openbrowser -->
        <test-method is-config="true" signature="login()[pri:0, instance:com.actitime.CScript2.CustomerModule22@25084a1e]" started-at="2022-09-02T21:13:52 IST" name="login" finished-at="2022-09-02T21:14:46 IST" duration-ms="53449" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Login]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="testcreateCustomer()[pri:0, instance:com.actitime.CScript2.CustomerModule22@25084a1e]" started-at="2022-09-02T21:14:46 IST" name="testcreateCustomer" finished-at="2022-09-02T21:15:03 IST" duration-ms="17435" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [CreatCustomer] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [CreatCustomer] but found []
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.actitime.CScript2.CustomerModule22.testcreateCustomer(CustomerModule22.java:46)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[testcreateCustomer]]>
            </line>
          </reporter-output>
        </test-method> <!-- testcreateCustomer -->
        <test-method is-config="true" signature="logout()[pri:0, instance:com.actitime.CScript2.CustomerModule22@25084a1e]" started-at="2022-09-02T21:15:03 IST" name="logout" finished-at="2022-09-02T21:15:07 IST" duration-ms="3044" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[logout]]>
            </line>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="closebrowser()[pri:0, instance:com.actitime.CScript2.CustomerModule22@25084a1e]" started-at="2022-09-02T21:15:07 IST" name="closebrowser" finished-at="2022-09-02T21:15:07 IST" duration-ms="1" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[closebrowser]]>
            </line>
          </reporter-output>
        </test-method> <!-- closebrowser -->
      </class> <!-- com.actitime.CScript2.CustomerModule22 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
